.github/workflows/conformance.yml

```diff
@@ -6,7 +6,6 @@ on:
       - main
   workflow_dispatch:
   pull_request_target:
```

Comment:
We can get rid of `pull_request_target` entirely, right?

Reply:
(i.e. it should just be `pull_request`)

Reply:
I think we can make it just `pull_request`.

Reply:
[d735a8d](https://github.com/sigstore/sigstore-python/pull/637/commits/d735a8d2b632c63da610aac3f80b9ffa8e9df6b8)